docs/source/deploying/prefixed-root.md

```diff
@@ -0,0 +1,68 @@
+---
+html_meta:
+  "description": "Prefix (non-root) deployment for Volto"
+  "property=og:description": "Prefix (non-root) deployment for Volto"
+  "property=og:title": "Prefixed (non-root) deployment"
+  "keywords": "Volto, deployment"
+---
+
+# Prefixed (non-root) deployment
+
```

Comment:
In general, one sentence per line. Docs are not code. This makes review of a complete thought much easier. Please update.